apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -68,12 +74,25 @@ const Input = <T extends FieldValues>({
 						))}
 					</div>
 				) : (
-					<input
-						className={getValidClassNames(styles["input-field"])}
-						{...field}
-						placeholder={placeholder}
-						type={type}
-					/>
+					<>
+						<input
+							className={getValidClassNames(styles["input-field"])}
+							{...field}
+							placeholder={placeholder}
+							type={type === "password" && isDisplayedValue ? "text" : type}
+						/>
+						{type === "password" && onToggle && (
+							<button
+								className={styles["toggle-password-icon"]}
+								onClick={onToggle}
+								type="button"
+							>
+								<Icon
+									name={isDisplayedValue ? IconName.CROSSED_EYE : IconName.EYE}
```

Comment:
iconName should be sent to Input component as a prop. Other inputs can also have icons